docs/platforms/flutter/overhead/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: SDK Overhead
+description: "Learn about Sentry's Flutter SDK overhead and how you can tailor your configuration to minimize it."
+sidebar_order: 7500
+---
+
+Adding new features or dependencies to your app incurs additional costs on resources - CPU time, memory usage, and network bandwidth, among other things. Sentry SDKs are no different. This document adds transparency to the possible overhead that using our SDK can add, and help you find the feature set and configurations that work best for you.
+
+## General Approach
+
+The SDK is designed to have minimal to no impact on user experience. To achieve this, we utilize an array of tools to continuously measure and optimize the performance of our implementations.
+
+We also employ various techniques to ensure we don't add strain on the system's resources along the hot path. On Mobile, this very often means that we offload processing steps, I/O, and other things to a background isolate, or we postpone processing to a later time if possible.
+
+If you find (for example via local Profiling, or using Sentry to improve the performance of your app) that the SDK does not operate within the guidelines mentioned below, please [open an issue](https://github.com/getsentry/sentry-dart/issues/new/choose) on our SDK repo and make sure you provide as much context as you can.
```

Comment:
Is the github link here supposed to go to the dart issues page or is there a separate one for flutter?

Reply:
the dart repo maintains both the dart and flutter SDK, similar to sentry-java which maintains the java and android sdk